addons/auth_passkeys/models/users_passkey.py

```diff
@@ -0,0 +1,90 @@
+from odoo import fields, models, api, _
+from odoo.addons.base.models.res_users import check_identity
+from odoo.exceptions import AccessDenied
+
+class UsersPasskey(models.Model):
+    _inherit = "res.users"
+
+    auth_passkeys_key_ids = fields.One2many("auth.passkeys.key", "create_uid")
+    password_login_disabled = fields.Boolean(string="Password Disabled",
```

Comment:
Might be an overkill feature for a first iteration. Besides the field name is really ugly :D.